tests/lib/Layout/Resolver/ConditionType/ResourceTypeTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Tests\Layout\Resolver\ConditionType;
+
+use Netgen\Layouts\Sylius\Layout\Resolver\ConditionType\ResourceType;
+use Netgen\Layouts\Sylius\Tests\Stubs\Product;
+use Netgen\Layouts\Sylius\Tests\Stubs\Taxon;
+use Netgen\Layouts\Sylius\Tests\Validator\SettingsValidatorFactory;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Validation;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+#[CoversClass(ResourceType::class)]
+final class ResourceTypeTest extends TestCase
+{
+    private ResourceType $conditionType;
+
+    private ValidatorInterface $validator;
+
+    protected function setUp(): void
+    {
+        $allowedResources = [
+            'Sylius\Component\Product\Model\ProductInterface' => 'product',
```

Comment:
`::class` constants here too.

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/11/commits/93ab79cc5e0f027f9222331aa0551e4358fa80db